config/initializers/mimemagic.rb

```diff
@@ -0,0 +1,6 @@
+require 'mimemagic'
+# require 'mimemagic/overlay'
+# file = File.expand_path('features/examples/shorter_lorem.docx', Rails.root)
+# MimeMagic.by_magic(File.open(file))
+
+MimeMagic.add('application/vnd.openxmlformats-officedocument.wordprocessingml.document', { extension: 'docx' })
```

Comment:
The overlay file has been reintroduced for version 0.3.9 so this may not be required any more.